source/customization_overview.rst

```diff
@@ -17,13 +14,35 @@ OnDemand's configuration is stored in the environment, which can be modified usi
 
    * **/etc/ood/config/apps/$APP/env**
 
+     - replace ``$APP`` above with the name of the app directory deployed to ``/var/www/ood/apps/sys``
      - Used to provide application specific config
      - ``env`` files do not override values set by prior methods.
 
    * **/etc/ood/config/apps/$APP/initializers/ood.rb**
 
+     - replace ``$APP`` above with the name of the app directory deployed to ``/var/www/ood/apps/sys``
      - Modify Rails application behavior `using Ruby code <https://guides.rubyonrails.org/configuring.html#using-initializer-files>`__. Since this is application code environment variables can be set, removed.
      - This method is specific to Ruby on Rails applications: Activejobs,
-       Dashboard, File Editor, and Job Composer. You can add multiple
```

Comment:
The file editor is a rails app. Did you mistake this for the file browser?

Reply:
This is intentional. I don't want to encourage people to add initializers to the file editor. I doubt anyone has. In 1.8 I intend to finally pull in Tyler's changes to make this a Sinatra (or simpler) app since Rails is unnecessary.